### DISCUSSION TITLE
Scraping player's agents data

### DISCUSSION THREAD

**Discussion by max-franceschi** [dcaribou/transfermarkt-datasets#95] [Q&A] [answered]
Hi David, 
Thank you for this awesome work. 

Transfermarkt displays players' agents (for those who have one). I assume it would be quite simple to scrape today's data. 
However, I was wondering if you had any idea about how to get these data for past years. Is there a way to scrape it or are you aware of a dataset that might include them? 
Otherwise, it might be an interesting addon to the players' table.

Max

**Comment by dcaribou** [comment, marked as answer]
Hey @max-franceschi, great idea!
I realised that the `player_agent` is actually being scraped already and hence it can easily be added to the dataset. I've created an issue for that [here](https://github.com/dcaribou/transfermarkt-datasets/issues/96).

The history however is more tricky. I'm currently limited to what can be scraped from transfermarkt, and I'm not aware of them keeping a history of player agents... If you find it in the website, I'm happy to take a look and see of that information can be incorporated.

**Reply by max-franceschi** [reply]
Thank you for your quick and purposeful feedback. Unfortunately, I am not aware of any history either... That's the tricky part. 

Looking forward to the addition of agents to your dataset. 

Thank you once again for the good job !

**Comment by Charlotte-br560** [comment]
Getting past years' player agent data from Transfermarkt can be tricky due to website changes. You could explore web archives like Wayback Machine.
